vision_agent_tools/models/florencev2.py

```diff
@@ -62,7 +62,7 @@ def _process_image(self, image: Image.Image) -> Image.Image:
     def _process_video(self, images: VideoNumpy) -> list[Image.Image]:
         return [self._process_image(Image.fromarray(arr)) for arr in images]
 
-    def __init__(self):
+    def __init__(self, device: Optional[Device] = Device.GPU) -> None:
```

Comment:
nit: we can use modern syntax for typing 
```suggestion
    def __init__(self, device: Device | None = Device.GPU) -> None:
```